### ISSUE TITLE
"A secret is required" problem

### ISSUE THREAD

**Issue by klauszhang** [sdm15stream2/serler#1]
A secret is required to generate an integrity hash for cookie session data. Use config.secret_token = "some secret phrase of at least 30 characters"in config/initializers/secret_token.rb

i think if we are using rails 4.2+ then it won't be a problem

Solution:

generate token
```
rake secret
```

create a new file and put it in position:
```
# config/initializers/secret_token.rb

# Be sure to restart your server when you modify this file.

# Your secret key for verifying the integrity of signed cookies.
# If you change this key, all old signed cookies will become invalid!
# Make sure the secret is at least 30 characters and all random,
# no regular words or you'll be exposed to dictionary attacks.
Serler::Application.config.secret_token = '_the token you got_'

```

**Comment by lpirl** [comment]
We could integrate this in README.md under 'initial steps' or something with a command like
<pre>
  cd to/serler/root/directory
  echo "Serler::Application.config.secret_token = '$(rake secret)'" > \
    config/initializers/secret_token.rb
</pre>

**Comment by klauszhang** [comment]
agree. or simply remove the lines about secret_token in .gitignore
```
config/initializers/secret_token.rb
config/secrets.yml
```

**Comment by lpirl** [comment]
There is a reason why it is called *secret* token - I would not recommend to share it since we probably use cookie-based authentication and the secret token protects against cookie tampering.

**Comment by klauszhang** [comment]
ok, I think this problem is solved and I will close the issue

**Comment by lpirl** [comment]
I included this in the Readme… ae24699
